src/cf-typography/src/atoms/date.less

```diff
@@ -0,0 +1,6 @@
+.a-date {
```

Comment:
Hmm, it's not a heading, it's used in the meta area here => http://www.consumerfinance.gov/about-us/blog/lo-que-necesita-saber-para-recibir-dinero-del-acuerdo-de-compensacion-con-honda-finance-por-cobrarles-de-mas-las-minorias/

Maybe we just need to be clearer where it should and shouldn't be used?